css/selectors/focus-visible-011.html

```diff
@@ -37,30 +37,34 @@
   <ul id="instructions">
     <li>Click "Click here and press right arrow.".</li>
     <li>Press the right arrow key.</li>
-    <li>If "Focus moves here." has a red background, then the test result is FAILURE.
+    <li>If the element has a red background, then the test result is FAILURE.
         If it has a green outline, then the test result is SUCCESS.</li>
   </ul>
   <br />
-  <button id="start" tabindex="0">Click here and press right arrow.</button>
-  <button id="next" tabindex="-1">Focus moves here.</button>
+  <div id="target" tabindex="0">Click here and press right arrow.</div>
   <script>
-    start.addEventListener('keydown', (e) => {
+    target.addEventListener("focus", () => {
+      const arrow_right = "\ue014";
+      test_driver.send_keys(target, arrow_right);
+    });
```

Comment:
This logic is actual part of the test, so I would move it into the `async_test`.

Reply:
True thing, moving it.